c7n/tags.py

```diff
@@ -457,6 +458,88 @@ def process_resource_set(self, vol_set, tag_keys):
             DryRun=self.manager.config.dryrun)
 
 
+class CopyTag(Action):
+    """ Create a new tag with identical value
+    """
+
+    schema = utils.type_schema(
+        'copy-tag',
+        old_key={'type': 'string'},
+        new_key={'type': 'string'})
+
+    permissions = ('ec2:CreateTags')
```

Comment:
this needs to be a set of permissions, here since there's only one string it's being interpreted as a single string, you need to change it to:

```python
permissions = ('ec2:CreateTags',)
```